### DISCUSSION TITLE
Only complete when there is one candidate

### DISCUSSION THREAD

**Discussion by hisnawi** [company-mode/company-mode#1189] [Q&A] [answered]
Is there an option to only complete when there is one candidate?
The reason is I am using helm-company which has fuzzy search and so I don't want to have both minibuffer and company completion popup. But I would prefer having company complete automatically when there is one candidate left, but not show the popup at all.

**Comment by dgutov** [comment]
You mean avoid showing the popup? You can edit `company-frontends` to remove that visualization.

**Comment by hisnawi** [comment, marked as answer]
Thanks dgutov, this did it:

`    (setq-default company-frontends '(company-preview-if-just-one-frontend))
`
